nautobot_ssot/integrations/infoblox/diffsync/models/nautobot.py

```diff
@@ -32,13 +32,25 @@ def process_ext_attrs(diffsync, obj: object, extattrs: dict):
                     diffsync.job.logger.warning(
                         f"Unable to find Location {attr_value} for {obj} found in Extensibility Attributes '{attr}'. {err}"
                     )
+                except TypeError as err:
+                    diffsync.job.logger.warning(
+                        f"Cannot set location values {attr_value} for {obj}. Multiple locations are assigned "
+                        f"in Extensibility Attributes '{attr}', but multiple location assignments are not "
+                        f"supported by Nautobot. {err}"
+                    )
             if attr.lower() == "vrf":
                 try:
                     obj.vrfs.add(diffsync.vrf_map[attr_value])
                 except KeyError as err:
                     diffsync.job.logger.warning(
                         f"Unable to find VRF {attr_value} for {obj} found in Extensibility Attributes '{attr}'. {err}"
                     )
+                except TypeError as err:
+                    diffsync.job.logger.warning(
```

Comment:
This is inaccurate for Nautobot 2.0. You can assign multiple VRFs to a Prefix through the `vrfs` attribute on the Prefix model. It's a M2M field so you need to use add() or set().

Reply:
fixed, multiple vrfs can now be assigned.